endpoints.php

```diff
@@ -2081,3 +2119,38 @@ function getKalturaVideoUrl($request)
         return false;
     }
 }
+
+function getKalturaAvailableLanguages($request)
+{
+    if (LOAD_KALTURA) {
```

Comment:
Thanks for bringing this up. I was also wondering about this. I think we show the "Use Kaltura" field in the VideoForm regardless of whether Kaltura is enabled on the backend, which means it is possible for these endpoints to be called when LOAD_KALTURA is false. However, I'm not sure how to respond when the user tries to edit an existing Kaltura video while Kaltura is not enabled on the backend (a rare situation, but possible).

Let's discuss this tomorrow, since this change may be better suited for 1168.

Reply:
Update from our discussion: This change is a good idea, but since we plan to keep Kaltura always enabled in production, I will create a separate PR off of 1168 to fix it.